### DISCUSSION TITLE
Long-overdue release update?

### DISCUSSION THREAD

**Discussion by gojo-limitless** [AnySoftKeyboard/AnySoftKeyboard#3979] [General]
I see plenty of commits to the project, but the Google Play Store says it was lasted updated a while back (which doesn't give confidence to users), and 2022 for F-Droid.

Any clarifications to the project's development by the developer? What should users expect?

**Comment by gojo-limitless** [comment]
@menny ? Google Play Store says - 2021 last updated, how can we recommend it to users with that date in mind? Any clarifications to that?

**Reply by menny** [reply]
I don't think the current version in production is valid for Android 13+. The latest beta version is valid.
There are a few issues I need to resolve before we can release a new version, but we're getting there.

**Reply by woj-tek** [reply]
What are the outstanding issues / how one could help? :-)

**Reply by menny** [reply]
https://github.com/AnySoftKeyboard/AnySoftKeyboard/milestone/94

**Comment by DI555** [comment]
@menny , could you switch on autobuildind in "Actions", please!
..it even may be copied into "Releases" after, or in major changes!
